ballerina-release-notes-0.990.0.md

```diff
@@ -390,6 +390,130 @@ Final variables
 Regular variables
 structure members
 
+## Endpoints, Services and Listeners
+
+### Endpoints
+
+In Ballerina, endpoints represent an external system that one or more workers interact with. There are two types of endpoints, Listener endpoints or client endpoints. 
+
+Previously, both these endpoints were modeled using two object definitions. One defined the life cycle of the endpoint, other defined the Actions of the endpoint. Additionally, endpoint keyword was used to represent endpoint instances and it had its own initialization and assignment syntax.
+
+With this release, the endpoints are declared as client objects. With this change following are removed from the language. 
+- endpoint declaration eg: `endpoint http:Listener helloWorldEP { port : 9090 }`
+- action invocation statement. (Only a terminology change)
```

Comment:
```suggestion
- Action invocation statement (only a terminology change).
```

Just checking: Is it terminology or syntax?

Reply:
it is a terminology change. The syntax remains the same. It is called as a remote method call, which explains later.